vmdb/spec/models/tenant_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+describe Tenant do
+  context "#default_tenant" do
+    before do
+      Tenant.seed
+    end
+
+    it { expect(described_class.default_tenant).to be }
+  end
+
+  context "#logo" do
+    let(:tenant) { described_class.create(:name => 'tenant1', :logo_file_name => "image.png") }
+
+    # NOTE: this currently returns a bogus url.
+    # it { expect(described_class.create(:name => 'tenant1').logo.url).to be_nil }
+
+    # for now, we are hard coding to uploads directory
+    it { expect(tenant.logo.url).to eq("/uploads/custom_logo.png") }
+
+    it { expect(tenant.logo.path).to eq(Rails.root.join("public/uploads/custom_logo.png").to_s) }
```

Comment:
Not a fan of description-less its personally, but considering...what is the long-form description of this one if there's an error?